.github/workflows/main.yml

```diff
@@ -63,3 +81,19 @@ jobs:
           git -c user.name='ci' -c user.email='ci' commit -m init
           git push -f -q https://git:${{ secrets.github_token }}@github.com/${{ github.repository }} HEAD:gh-pages
         if: github.event_name == 'push' && github.event.ref == 'refs/heads/master'
+
+  msrv:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-14, windows-latest]
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Rust (rustup)
+      run: rustup toolchain install nightly --no-self-update --profile minimal
+      shell: bash
+
+    - uses: taiki-e/install-action@cargo-hack
+    - uses: Swatinem/rust-cache@v2
+
+    - run: cargo hack check --lib --rust-version --ignore-private
```

Comment:
This needs a `-locked` flag

Reply:
Updated tests and msrv to use `--locked`